src/components/Problems/Discussion.tsx

```diff
@@ -121,9 +122,14 @@ export const Discussion: FC<DiscussionProps> = ({problemId, invalidateSeriesQuer
 
               return (
                 <Stack sx={comment.state !== CommentState.Published ? {color: 'gray'} : {}} key={comment.id}>
-                  <Typography variant="h3" component="span">
-                    {comment.posted_by_name}
-                  </Typography>
+                  <Stack gap={1} direction="row" justifyContent={'space-between'}>
```

Comment:
gap={1} je uz asi zbytocny, ale mozno aj nie - na malej obrazovke, ked by sa to uz tlacilo na seba, tak sa browser bude snazit aspon kus to oddelit

Reply:
ja netusim co to robi to bol len copy paste :joy: